wvdecrypter/cdm/media/cdm/cdm_adapter.cc

```diff
@@ -307,6 +318,8 @@ void CdmAdapter::CloseSession(uint32_t promise_id,
   const char* session_id,
   uint32_t session_id_size)
 {
+  exit_thread_flag = true;
+  std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
This value appears arbitrary. What exactly is this waiting for?

This value could very well work on one system but not on another. It would be better to continue once a condition is satisfied.

Reply:
Same as above.